src/stories/StyledTabContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactNode } from 'react';
+import StyledTab from './StyledTab';
+
+interface StyledTabContainerProps {
+  tabs: Array<TabProps>;
+  openTabOnLoad?: number;
+}
+
+type TabProps = {
+  title: string;
+  deactivated: boolean;
+  content: ReactNode | undefined;
+  type?: TabType;
+};
+
+export enum TabType {
+  BUY = 'BUY',
+  SOON = 'COMING SOON',
+}
+
+export default function StyledTabContainer({ tabs, openTabOnLoad = 0 }: StyledTabContainerProps) {
+  const [openTab, setOpenTab] = React.useState(openTabOnLoad);
```

Comment:
`React` can be omitted:
`const [openTab, setOpenTab] = useState(openTabOnLoad);`

Reply:
changed.